http/src/main/resources/reference.conf

```diff
@@ -627,7 +631,7 @@ pubsub {
     }
     createApp {
       interval = 10 seconds
-      max-attempts = 120 # 10 seconds * 120 = 20 min
+      max-attempts = 60 # (10 seconds + ~20 second http4s retry) * 60 =~ 30 min
```

Comment:
Thinking another option may be to call `interruptAfter(30 minutes)` on the Stream 🤔